django/core/management/__init__.py

```diff
@@ -317,6 +317,10 @@ def execute(self):
             settings.INSTALLED_APPS
         except ImproperlyConfigured as exc:
             self.settings_exception = exc
+        except ModuleNotFoundError as exc:
```

Comment:
The tests are failing with:

```
+ flake8
./django/core/management/__init__.py:320:16: F821 undefined name 'ModuleNotFoundError'
```

and rightly so. I think the exception is removed in Python 3 from what I can see on Google.

Reply:
I tried running flake8 in python3.6, It does not give any issues because it seems to exist in python3.6 but `ModuleNotFoundError` did not exist in python3.5, so removing `ModuleNotFoundError` now. It works the same with only `ImportError` and `ImproperlyConfigured`.

Reply:
you still haven't pushed a commit removing it?

Reply:
no I haven't. I'll push with the test today.

Reply:
Just checked, it's safe to only catch `ImportError` as `ModuleNotFoundError` is a subclass of it: https://docs.python.org/3/library/exceptions.html#ModuleNotFoundError